Framework/PythonInterface/plugins/algorithms/MRGetTheta.py

```diff
@@ -27,23 +27,33 @@ def PyInit(self):
         self.declareProperty("AngleOffset", 0.,FloatBoundedValidator(lower=0.), "Angle offset (rad)")
         self.declareProperty("UseSANGLE", False, doc="If True, use SANGLE as the scattering angle. If False, use DANGLE.")
         self.declareProperty("SpecularPixel", 0., doc="Pixel position of the specular reflectivity [optional]")
-        self.declareProperty("Theta", 0., direction=Direction.Output, doc="Scattering angle theta [rad]")
-
+        self.declareProperty("DirectPixelOverwrite", Property.EMPTY_DBL, doc="DIRPIX overwrite value")
+        self.declareProperty("DAngle0Overwrite", Property.EMPTY_DBL, doc="DANGLE0 overwrite value (degrees)")
+        self.declareProperty("Theta", 0., direction=Direction.Output, doc="Scattering angle theta (rad)")
         return
 
     def PyExec(self):
         """ Main execution body """
         _w=self.getProperty("Workspace").value
 
         angle_offset = self.getProperty("AngleOffset").value
+        dirpix_overwrite = self.getProperty("DirectPixelOverwrite").value
+        dangle0_overwrite = self.getProperty("DAngle0Overwrite").value
+        use_sangle = self.getProperty("UseSANGLE").value
 
-        if self.getProperty("UseSANGLE").value:
+        if use_sangle:
             theta = self.read_log(_w, 'SANGLE', target_units='rad', assumed_units='deg')
         else:
             dangle = self.read_log(_w, 'DANGLE', target_units='rad', assumed_units='deg')
-            dangle0 = self.read_log(_w, 'DANGLE0', target_units='rad', assumed_units='deg')
+            if dangle0_overwrite == Property.EMPTY_DBL:
+                dangle0 = self.read_log(_w, 'DANGLE0', target_units='rad', assumed_units='deg')
+            else:
+                dangle0 = dangle0_overwrite * math.pi / 180.0
```

Comment:
Use `numpy.radians()` instead

Reply:
I'd rather keep it simple since there's a few simple unit conversions throughout the algorithm and I'd like to keep the same approach for all of them.

Reply:
OK